<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ValidateLogLengthDialog</class>
    <widget class="QDialog" name="ValidateLogLengthDialog">
        <property name="windowTitle">
            <string>Confirm Actual Length Used</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="expectedLabel">
                            <property name="text">
                                <string>Expected Length (in):</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" name="expectedLengthValueLabel">
                            <property name="text">
                                <string>0.0</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="actualLabel">
                            <property name="text">
                                <string>Actual Length Used (in):</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QDoubleSpinBox" name="ValidateLogLengthSpin">
                            <property name="minimum">
                                <double>0.0</double>
                            </property>
                            <property name="maximum">
                                <double>1000.0</double>
                            </property>
                            <property name="value">
                                <double>0.0</double>
                            </property>
                            <property name="singleStep">
                                <double>0.25</double>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <layout class="QHBoxLayout" name="buttonLayout">
                    <item>
                        <spacer name="spacer">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>40</width>
                                    <height>20</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QPushButton" name="cancelButton">
                            <property name="text">
                                <string>Cancel</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="okButton">
                            <property name="text">
                                <string>OK</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources />
    <connections />
</ui>